modules/ppcp-wc-gateway/src/Gateway/PayUponInvoice/PayUponInvoice.php

```diff
@@ -518,7 +518,8 @@ public function register_assets(): void {
 				'ppcp-pay-upon-invoice',
 				trailingslashit( $this->module_url ) . 'assets/js/pay-upon-invoice.js',
 				array(),
-				$this->asset_version
+				$this->asset_version,
+				true
```

Comment:
Is there a reason why `in_footer` was not `true`?

Reply:
These is no reason, I forgot to pass it here.